tasks/main.yml

```diff
@@ -21,6 +21,10 @@
     dest: "{{ jekyll_build_sourcedir }}/{{ jekyll_build_git_repo | basename }}"
     force: "{{ jekyll_build_force_git }}"
 
+- name: install bundler
+  become: yes
+  shell: gem install bundler -v 1.16.1
```

Comment:
2 questions:
1. Is there a `gem` command in ansible? 
2. I think we should parametrize the bundler version :)